CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to STEV
+
+Thank you for considering making contributions to the project! A brief outline
```

Comment:
2nd Sentence -> This document contains a brief outline of the guidelines for contributing.